src/cli/main.pl

```diff
@@ -1221,67 +1227,56 @@
 run_command(doc,insert,[Path], Opts) :-
     super_user_authority(Auth),
     create_context(system_descriptor{}, System_DB),
-    option(author(Author), Opts),
-    option(message(Message), Opts),
-    option(graph_type(Graph_Type), Opts),
     option(data(Data), Opts),
-    option(full_replace(Full), Opts),
 
     api_report_errors(
         insert_documents,
         (   var(Data)
-        ->  with_memory_file(cli:doc_insert_memory_file(System_DB, Auth, Path, Graph_Type, Author, Message, Full, Ids))
+        ->  with_memory_file(cli:doc_insert_memory_file(System_DB, Auth, Path, Ids, Opts))
         ;   open_string(Data, Stream),
-            doc_insert_stream(System_DB, Auth, Path, Graph_Type, Author, Message, Full, Ids, Stream)
+            doc_insert_stream(System_DB, Auth, Path, Ids, Opts, Stream)
         )
     ),
     length(Ids, Number_Inserted),
     (   Number_Inserted = 1
-    ->  format(current_output, "Document inserted~n", [])
-    ;   format(current_output, "Documents inserted: ~d~n", [Number_Inserted])
+    ->  format(current_output, "Document inserted ~q~n", [Ids])
```

Comment:
I'd actually prefer the reverse, where you get a `--quiet` but it is in fact really important in practice because Ids can be randomly generated after which point you haven't a clue what you've added.

Reply:
We can add `--quiet` in a subsequent PR.